src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -34,6 +35,8 @@
     public static final String VALID_EMAIL_BOB = "bob@example.com";
     public static final String VALID_ADDRESS_AMY = "Block 312, Amy Street 1";
     public static final String VALID_ADDRESS_BOB = "Block 123, Bobby Street 3";
+    public static final String VALID_BIRTHDATE_AMY = "2000-01-01";
```

Comment:
`VALID_BIRTHDATE_AMY` needs to be updated to `VALID_BIRTH_DATE_AMY`? Same as other constants.